pykern/pkjson.py

```diff
@@ -60,16 +72,16 @@ def dump_pretty(obj, filename=None, pretty=True, **kwargs):
     return res
 
 
-def dump_bytes(obj, **kwargs):
-    """Formats as json as bytes for network transfer
+def dump_str(obj, **kwargs):
+    """Formats as a compact json as string
 
     Args:
         obj (object): any Python object
         kwargs (object): other arguments to `dump_pretty`
     Returns:
-        bytes: (unsorted) formatted JSON
+        str: (unsorted) formatted JSON
     """
-    return dump_pretty(obj, pretty=False, **kwargs).encode(ENCODING)
+    return dump_pretty(obj, pretty=False, **kwargs)
```

Comment:
What is dump_str() for? Is it also for network transfers or just a generally useful conversion?

Reply:
Nvm, covered it in the software meeting